<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>86</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>86</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>86</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Traitement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/images_ressource/logo.ico</normaloff>:/images_ressource/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_traiter">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>traiter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_parcourir">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>parcourir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_terminer">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>termin√© !</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_traitement">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>traitement en cours...</string>
    </property>
   </widget>
   <widget class="FileEdit" name="fileEdit_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileEdit</class>
   <extends>QLineEdit</extends>
   <header>main.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_parcourir</tabstop>
  <tabstop>pushButton_traiter</tabstop>
 </tabstops>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
